### ISSUE TITLE
Errors installing npm module during yo mobile scaffolding

### ISSUE THREAD

**Issue by mashcode** [yeoman/generator-mobile#76]
Not sure if this is npm or yo related or even an "issue"?!...but it also may be causing gulp to later fail on  "Error: Cannot find module"
...
✔ You are all set now. Happy coding!
npm ERR! Darwin 14.1.0
npm ERR! argv "node" "/usr/local/bin/npm" "install"
npm ERR! node v0.12.0
npm ERR! npm  v2.5.1

npm ERR! null == true
npm ERR! 
npm ERR! If you need help, you may report this error at:
npm ERR!     <http://github.com/npm/npm/issues>

> ws@0.5.0 install /Users/marc/Sites/NYCompost/site/node_modules/browser-sync/node_modules/socket.io/node_modules/engine.io/node_modules/ws
> (node-gyp rebuild 2> builderror.log) || (exit 0)

  CXX(target) Release/obj.target/bufferutil/src/bufferutil.o
  SOLINK_MODULE(target) Release/bufferutil.node
  SOLINK_MODULE(target) Release/bufferutil.node: Finished
  CXX(target) Release/obj.target/validation/src/validation.o
  SOLINK_MODULE(target) Release/validation.node
  SOLINK_MODULE(target) Release/validation.node: Finished
npm WARN deprecated tap-consumer@0.0.1: this is merged into the 'tap' module now
npm WARN deprecated tap-results@0.0.2: this is merged into the 'tap' module now

...

> ws@0.4.31 install /Users/marc/Sites/NYCompost/site/node_modules/browser-sync/node_modules/socket.io/node_modules/socket.io-client/node_modules/engine.io-client/node_modules/ws
> (node-gyp rebuild 2> builderror.log) || (exit 0)

  CXX(target) Release/obj.target/bufferutil/src/bufferutil.o

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/marc/Sites/NYCompost/site/npm-debug.log

**Comment by sindresorhus** [comment]
> npm ERR! If you need help, you may report this error at:
npm ERR! http://github.com/npm/npm/issues

**Comment by crhym3** [comment]
@mashcode It fails at the last step, `npm install`.

It has nothing to do with this generator but unfortunately it happens from time to time.

Try removing node_modules and running `npm install` manually.

**Comment by mashcode** [comment]
Thanks Alex, I predict a node/npm installfest in my future...:P

email marc@marcshifflett.com | mobile 917-570-7079

**Comment by mashcode** [comment]
I might as well close and revisit since short of building node I've settled on the node binary. Stable without having to hijack npm.

**Summary** The yo mobile generator scaffolds but fails at module installs, a manual npm install "seems" to work but with warnings and gulp serves up this: 
throw err;
          ^
Error: Cannot find module 'core-util-is' 

I destroyed the log so if you want to see it let me know. Another step closer to becoming markdown ######luddite. 
Cheers!
